canopen/objectdictionary/eds.py

```diff
@@ -463,7 +463,7 @@ def export_record(var, eds):
     for rate in od.device_information.allowed_baudrates.union(
             {10e3, 20e3, 50e3, 125e3, 250e3, 500e3, 800e3, 1000e3}):
         eds.set(
-            "DeviceInfo", "BaudRate_%i" % (rate/1000),
+            "DeviceInfo", f"BaudRate_{rate//1000}",
```

Comment:
Is truncating division here really the same as the previous implicit conversion to int? (or was there an exception for bitrates not divisible by 1000?)

Reply:
It is the same

```
>>> "%i" % (2900/1000)     # It floors the value
'2'
>>> f"{2900/1000}"     # Don't want this
'2.9'
>>> f"{2900//1000}"    # f-string equivalent
'2'
>>> f"{2900/1000:d}"    # Giving float to :d doesn't work
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Unknown format code 'd' for object of type 'float'
```